extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -115,6 +119,22 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /**
+   * Shows the controls and then fades them away.
+   */
+  hintControls() {
+    if (this.showControls_ || !this.isInViewport()) {
+      return;
+    }
+    this.getVsync().mutate(() => {
+      const className = 'amp-carousel-button-start-hint';
+      this.element.classList.add(className);
+      timer.delay(() => {
+        this.element.classList.remove(className);
```

Comment:
Both add and remove should be in mutate context. Timer doesn't give us that. So, thinking this through a bit more - we should do `remove` to `deferMutate`. Why? Because removing a class in this case is not critical and thus deferring it is a good idea. So, it'll be:
```
timer.delay(() => {
  this.deferMutate(() => this.element.classList.remove(className));
}, ...);
```